javascript-forms/form-values.js

```diff
@@ -0,0 +1,14 @@
+var contactform = document.querySelector('#contact-form');
+
+function handleSubmit(event) {
+  event.preventDefault();
+  var formresults = {
+    name: contactform.elements.name.value,
+    email: contactform.elements.email.value,
+    message: contactform.elements.message.value
+  };
+  console.log(formresults);
+  document.querySelector('#contact-form').reset();
```

Comment:
You have already selected the form, you don't need to re query for it. You can just use `contactForm.reset()`